src/units/frame.cpp

```diff
@@ -45,7 +52,11 @@ progressive_string::progressive_string(const std::string & data,int duration) :
 		for(tmp=first_pass.begin(); tmp != first_pass.end(); ++tmp) {
 			std::vector<std::string> second_pass = utils::split(*tmp,':');
 			if(second_pass.size() > 1) {
-				data_.push_back(std::pair<std::string,int>(second_pass[0],std::stoi(second_pass[1])));
+				try {
+					data_.push_back(std::pair<std::string,int>(second_pass[0],std::stoi(second_pass[1])));
+				} catch(std::invalid_argument) {
+					ERR_NG << "Invalid time in unit animation: " << second_pass[0] << ", " << second_pass[1] << "\n";
```

Comment:
No need to output `second_pass[0]` here - if you look, it's not trying to be converted to an integer.